packages/manager/modules/iplb/src/frontends/index.js

```diff
@@ -0,0 +1,89 @@
+
+import IpLoadBalancerDashboardHeaderCtrl from '../header/iplb-dashboard-header.controller';
+import IpLoadBalancerFrontendsCtrl from './iplb-frontends.controller';
+import IpLoadBalancerFrontendDeleteCtrl from './delete/iplb-frontends-delete.controller';
+import IpLoadBalancerFrontendsEditCtrl from './iplb-frontends-edit.controller';
+import IpLoadBalancerFrontendPreviewCtrl from './preview/iplb-frontends-preview.controller';
+import IpLoadBalancerFrontendsService from './iplb-frontends.service';
+
+import IplbFrontendsEditTemplate from './iplb-frontends-edit.html';
+import IplbFrontendsTemplate from './iplb-frontends.html';
+import IplbHeaderTemplate from '../header/iplb-dashboard-header.html';
+
+const moduleName = 'ovhManagerIplbFrontends';
+
+angular
+  .module(moduleName, [
+    'ui.router',
+  ])
+  .config(($stateProvider) => {
```

Comment:
```suggestion
  .config(/* @ngInject */($stateProvider) => {
```
Also shoudn't the routing be extracte ?